source/common/http/conn_manager_impl.cc

```diff
@@ -601,12 +601,10 @@ void ConnectionManagerImpl::ActiveStream::completeRequest() {
     filter_manager_.streamInfo().setResponseFlag(
         StreamInfo::ResponseFlag::DownstreamConnectionTermination);
   }
-  if (connection_manager_.codec_->protocol() < Protocol::Http2) {
-    // For HTTP/2 there are still some reset cases where details are not set.
-    // For HTTP/1 there shouldn't be any. Regression-proof this.
+  // TODO(danzh) bring HTTP/3 to parity here.
```

Comment:
I feel like we are accumulating a large number of this type of thing. Should we be throwing these into one of the QUIC tracking issues? cc @danzh2010?

Reply:
What are the response details for? I don't think QUICHE take such a string, it maybe something else in QUICHE.

It makes sense to track codec parity between H2 and H3.

Reply:
Dan - just like gfe response code details, covers the reason the stream is closed.

Reply:
regarding accumulating TODOs I think it's fine if Dan would like a tracking issue, but I think the TODO that it's expected for HTTP/3 is useful

Reply:
My concern is we are accumulating a lot of these TODOs and I'm concerned we are going to wind up with subtle bugs if we don't carefully track what needs fixing either with a special TODO search string or issue.